docs/cql/dml.rst

```diff
@@ -202,7 +202,7 @@ It is also possible to “group” ``CLUSTERING COLUMNS`` together in a relation
        AND (blog_title, posted_at) > ('John''s Blog', '2012-01-01')
 
 will request all rows that sort after the one having “John's Blog” as ``blog_title`` and '2012-01-01' for ``posted_at``
-in the clustering order. In particular, rows having a ``post_at <= '2012-01-01'`` will be returned as long as their
```

Comment:
@annastuchlik the email in the Co-authored-by: tag (see changelog) was mangled by github.

Reply:
I'm confused. What should I do to sort out anything that's wrong?

Reply:
Look like my rebase attampt was wrong.
Closing this PR< I will send a v2

Reply:
new PR https://github.com/scylladb/scylladb/pull/12596